index.js

```diff
@@ -38,6 +38,51 @@ searchBtnCmp.addEventListener("click", function() {
             tempUnitCmp.innerHTML = "&#8451";
             newWindVal.innerHTML = Math.round(data.wind.speed);
             weatherWindCmp.appendChild(newWindVal);
+            let weatherIcon = data.weather[0].icon;
+            let icons = new Skycons({ "color": "white" });
+            icons.set("weatherIcon", Skycons.CLEAR_DAY);
```

Comment:
you have a lot of duplication here. Please put all possible statuses in an array and then do this set in for-loop.